src/Components/Container/Container.js

```diff
@@ -17,6 +18,11 @@ export class Container extends Component {
     this.props.navTool.navigate('routinePage');
   }
 
+  routeToDetails = (id) => {
```

Comment:
Consider changing this name.  While route is technically a verb, it is often confused with a noun.